script/test/fixtures/buildargs/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "2"
+
+services:
+    foo:
+        build: 
+          context: "./build"
+          args:
+            NAME: web
+        command: "sleep 3600"
+
+    foo1:
+            build:
+              context: "./build"
+              args:
+                 - NAME=web
+                 - foo
```

Comment:
@cdrage , this type is not supported in build args, and as `buildargs` is of `map[string]*string`, it throws error,

```
ERRO Failed to unmarshall: Cannot unmarshal '<nil>' to type <nil> into a string value

```